Lib9c.Policy/Policy/MaxTransactionsPerBlockPolicy.cs

```diff
@@ -17,7 +17,13 @@ private MaxTransactionsPerBlockPolicy(int defaultValue)
         public static IVariableSubPolicy<int> Default =>
             new MaxTransactionsPerBlockPolicy(int.MaxValue);
 
-        public static IVariableSubPolicy<int> Mainnet =>
+        public static IVariableSubPolicy<int> Odin =>
+            Default
+                .Add(new SpannedSubPolicy<int>(
+                    startIndex: 0,
+                    value: BlockPolicySource.MaxTransactionsPerBlock));
```

Comment:
Just note: the `BlockPolicySource.MaxTransactionsPerBlock` value should be in this `MaxTransactionsPerBlockPolicy.cs` file. 🤔